src/Sylius/Bundle/CoreBundle/Resources/views/Email/accountVerification.html.twig

```diff
@@ -0,0 +1,22 @@
+{% extends '@SyliusCore/Email/layout.html.twig' %}
+
+{% block subject %}
+    {{ 'sylius.email.verification_token.subject'|trans({}, null, localeCode) }}
+{% endblock %}
+
+{% block content %}
+    {% set url = channel.hostname is not null ? 'http://' ~ channel.hostname ~ path('sylius_shop_user_verification', { 'token': user.emailVerificationToken}) : url('sylius_shop_user_verification', { 'token': user.emailVerificationToken}) %}
```

Comment:
We need to decouple this email from the shop bundle

```suggestion
{% if sylius_bundle_loaded_checker('SyliusShopBundle') %}
    {% set url = channel.hostname is not null ? 'http://' ~ channel.hostname ~ path('sylius_shop_user_verification', { 'token': user.emailVerificationToken}) : url('sylius_shop_user_verification', { 'token': user.emailVerificationToken}) %}
{% endif %}
```

Reply:
Also, this would come up if we would have a test for this email. We should write PHPUnit for it. Take a look into `\Sylius\Bundle\CoreBundle\Tests\Mailer\OrderEmailManagerTest` for inspiration